src/main/resources/templates/repo/kinesis-log-streams.json

```diff
@@ -14,21 +14,62 @@
 		"bufferFlushInterval": 900,
 		"tableDescriptor": {
 			"name": "fileDownloadsRecords",
-			"columns": {
-				"userId": "bigint",
-				"timestamp": "timestamp",
-				"projectId": "bigint",
-				"fileHandleId": "string",
-				"associateType": "string",
-				"associateId": "string",
-				"stack": "string",
-				"instance": "string"
-			},
-			"partitionKeys": {
-				"year": "string",
-				"month": "string",
-				"day": "string"
-			}
+			"description" : "The table contain records of all downloads of Synapse. The events are recorded only after the pre-signed url for requested download entity is generated.",
+			"columns": [{
+				"name": "userId",
+				"type" : "bigint",
+				"comment" : "The id of the user who downloaded the file."
+			},
+			{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the download event is pushed to the queue for recording, after generating the Pre-signed url."

```

Comment:
pre-signed url (no capitalization)

Please see my comments `etl-jobs-config.json`, above, for the column descriptions duplicated in this file.